resources/views/hardware/view.blade.php

```diff
@@ -919,6 +919,10 @@
                                                 </li>
                                             @endif
 
+                                            @if((isset($asset->assignedTo)) && ($asset->assignedTo->department!=''))
```

Comment:
I'd probably change this to:

`@if((isset($asset->assignedTo)) && ($asset->assignedTo->department))`

so that we're making sure that the dept actually exists